recipe_batches/recipe_batches.py

```diff
@@ -3,12 +3,37 @@
 import math
 
 def recipe_batches(recipe, ingredients):
-  pass 
+  recipeValue = list(recipe.values())
+  ingredientsQuantity = list(ingredients.values())
```

Comment:
Is it worth converting these into arrays to later access by index?

perhaps leaving them as objects and using corresponding iteration such as `for items in recipe` and `if items in ingredients` could clean up the readability.